<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="chooseInputLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Choose &amp;Input CSV file</string>
          </property>
          <property name="buddy">
           <cstring>inputLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="inputLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="chooseModelLabel">
          <property name="text">
           <string>Choose Model</string>
          </property>
          <property name="buddy">
           <cstring>modelLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="chooseSerialPortLabel">
          <property name="text">
           <string>Serial &amp;Port</string>
          </property>
          <property name="buddy">
           <cstring>serialPortComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="chooseOutputFolderLabel">
          <property name="text">
           <string>Choose &amp;Output Folder</string>
          </property>
          <property name="buddy">
           <cstring>outputFolderLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="chooseLabLabel">
          <property name="text">
           <string>Choose &amp;Lab</string>
          </property>
          <property name="buddy">
           <cstring>labNameComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="browseInputButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QComboBox" name="labNameComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="serialPortComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="connectDisconnectSerialButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="outputFolderLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="browseOutputButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="refreshSerialPortsButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="modelLineEdit"/>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="useDefaultModelCheckbox">
          <property name="text">
           <string>Use Default Model</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="browseModelButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="startStopButton">
        <property name="text">
         <string>Start Processing</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="logSplitterMajor">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="processingLogLabel">
       <property name="text">
        <string>Processing Log</string>
       </property>
      </widget>
      <widget class="QSplitter" name="logSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTextBrowser" name="logTextBrowser"/>
       <widget class="QSplitter" name="logButtonsSplitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="clearLogButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveLogButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="progressLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="progressLabel">
        <property name="text">
         <string>Progress</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lastLogTextLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>labNameComboBox</tabstop>
  <tabstop>modelLineEdit</tabstop>
  <tabstop>useDefaultModelCheckbox</tabstop>
  <tabstop>browseModelButton</tabstop>
  <tabstop>inputLineEdit</tabstop>
  <tabstop>browseInputButton</tabstop>
  <tabstop>outputFolderLineEdit</tabstop>
  <tabstop>browseOutputButton</tabstop>
  <tabstop>serialPortComboBox</tabstop>
  <tabstop>connectDisconnectSerialButton</tabstop>
  <tabstop>refreshSerialPortsButton</tabstop>
  <tabstop>startStopButton</tabstop>
  <tabstop>clearLogButton</tabstop>
  <tabstop>saveLogButton</tabstop>
  <tabstop>logTextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
